components/src/core/UserMenu/UserMenu.tsx

```diff
@@ -33,11 +33,18 @@ const useStyles = makeStyles(() =>
         noCursor: {
             cursor: 'inherit',
         },
+        bottomSheet: {
+            width: '100%',
+            maxWidth: 600,
```

Comment:
If you set `useBottomSheetAt = theme.breakpoints.values.sm,` then I expect every "600" in your code to get changed